extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/MultiValueScrutinizer.java

```diff
@@ -3,15 +3,36 @@
 import org.openrefine.wikidata.qa.QAWarning;
 import org.openrefine.wikidata.updates.ItemUpdate;
 import org.wikidata.wdtk.datamodel.interfaces.PropertyIdValue;
+import org.wikidata.wdtk.datamodel.interfaces.Snak;
+import org.wikidata.wdtk.datamodel.interfaces.SnakGroup;
 import org.wikidata.wdtk.datamodel.interfaces.Statement;
+import org.wikidata.wdtk.datamodel.interfaces.Value;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class MultiValueScrutinizer extends EditScrutinizer {
 
     public static final String new_type = "multi-valued-property-is-required-for-new-item";
     public static final String existing_type = "multi-valued-property-is-required-for-existing-item";
+    public static String MULTI_VALUE_CONSTRAINT_QID = "Q21510857";
+    public static String MULTI_VALUE_CONSTRAINT_STATUS = "P2316";
+
+    class MultivalueConstraint {
```

Comment:
Is this inner class actually used in the scrutinizer?

Reply:
Actually, no. Because it has just one parameter as constraint status, which we are not currently using to raise warnings. Should we consider that? then only this class makes any sense.

Reply:
I think it the general structure of scrutinizers should be like this:
- fetch all constraint statements of a given type via the fetcher
- map these to constraint definitions (`.map(statement -> new MultivalueConstraint(statement))`)
- for each constraint definition, run the scrutinizing code

So in this case the inner class would still be "used": the scrutinizing code would just not rely on it to fetch parameters for now. In the future, when we add support for exceptions, this class will still be useful :) Or we could replace it simply by the base class that fetches generic constraint parameters, too.